Formula/liquidprompt.rb

```diff
@@ -10,21 +10,27 @@ class Liquidprompt < Formula
     sha256 cellar: :any_skip_relocation, all: "407ac67632d4c74525162eedfbc1033ca4ea69ca70a8ce225531d09c35d195eb"
   end
 
+  doc_url = "https://liquidprompt.readthedocs.io/en/stable/config.html"
+
   def install
-    share.install "liquidpromptrc-dist"
     share.install "liquidprompt"
   end
 
-  def caveats
+  define_method :caveats do
+    if build.head?
+      doc_url = "https://liquidprompt.readthedocs.io/en/latest/config.html"
+      head_note = " (for the HEAD version)"
+    end
+
     <<~EOS
       Add the following lines to your bash or zsh config (e.g. ~/.bash_profile):
         if [ -f #{HOMEBREW_PREFIX}/share/liquidprompt ]; then
           . #{HOMEBREW_PREFIX}/share/liquidprompt
         fi
 
       If you'd like to reconfigure options, you may do so in ~/.liquidpromptrc.
-      A sample file you may copy and modify has been installed to
-        #{HOMEBREW_PREFIX}/share/liquidpromptrc-dist
+      Configuration documentation#{head_note} can be found here:
+        #{doc_url}
```

Comment:
I think we should just remove this if we'll just refer to upstream docs. Caveats are only for Homebrew documentation

Reply:
It's not obvious how to set up or configure the tool, which is why we'd like to put that in front of the user at install time. In the future we may again be able to generate and provide a sample file based on the current state of the documentation.

Reply:
So, most people do not read the caveats and even less people will if we start using it for all kinds of documentation.